pkg/validate/serialize.go

```diff
@@ -0,0 +1,44 @@
+package validate
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/ghodss/yaml"
+
+	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+)
+
+//Verify takes in name of the yaml file to be validated, reads it, and calls the unmarshal function on the rawYaml.
+func Verify(yamlFileName string) error {
+	rawYaml, err := ioutil.ReadFile(yamlFileName)
+	if err != nil {
+		return fmt.Errorf("Error in reading yaml file:   #%s ", err)
```

Comment:
Would be good if you specified the file name in the error message. It will help later when there are multiple yamls to validate at once and we want to know which caused an error

Reply:
Done.